sonic_sku_create/sonic_sku_create.py

```diff
@@ -0,0 +1,329 @@
+#! /usr/bin/python
+"""
```

Comment:
Move this script into /script directory?
Add it into setup.py?
Add a unit test?

Reply:
Moved into scripts directory and added it to setup.py